docs/conf.py

```diff
@@ -65,3 +66,6 @@ def setup(app):
 
 
 # endregion
+
+# Client-side page redirects.
+rediraffe_redirects = "redirects.txt"
```

Comment:
Redirects depend on [Rediraffe](https://github.com/sphinx-doc/sphinxext-rediraffe). That needs adding [to the project](https://github.com/canonical/snapcraft/blob/6fa334402a1ebaa6ff58064d49f1f46fd01fb404/pyproject.toml#L93) and [to this conf](https://github.com/canonical/snapcraft/blob/6fa334402a1ebaa6ff58064d49f1f46fd01fb404/docs/conf.py#L79).